crates/client/block_import/src/verify_apply.rs

```diff
@@ -293,3 +294,700 @@ fn block_hash(
 
     Ok((block_hash, header))
 }
+
+#[cfg(test)]
+mod verify_apply_tests {
+    use super::*;
+    use mc_db::DatabaseService;
+    use mp_block::header::{GasPrices, L1DataAvailabilityMode};
+    use mp_block::Header;
+    use mp_chain_config::{ChainConfig, StarknetVersion};
+    use mp_receipt::{
+        DeclareTransactionReceipt, DeployAccountTransactionReceipt, DeployTransactionReceipt, InvokeTransactionReceipt,
+        L1HandlerTransactionReceipt,
+    };
+    use mp_state_update::StateDiff;
+    use mp_transactions::{
+        DeclareTransactionV1, DeployAccountTransactionV3, DeployTransaction, InvokeTransactionV1, L1HandlerTransaction,
+    };
+    use starknet_api::{core::ChainId, felt};
+    use std::sync::Arc;
+    use tempfile::TempDir;
+
+    /// Creates a dummy UnverifiedHeader for testing purposes.
+    ///
+    /// This function generates an UnverifiedHeader with predefined values,
+    /// useful for creating consistent test scenarios.
+    fn create_dummy_unverified_header() -> UnverifiedHeader {
```

Comment:
please include the dummy functions directly in the structure declaration file to enable easy reuse throughout the codebase. For example, the create_dummy_unverified_header function could be modified to take a protocol_version parameter

Reply:
when you say structure declaration file, what does it mean? should have these helpers in a different tesl util file?